pkg/nsx/services/subnet/builder.go

```diff
@@ -25,11 +25,11 @@ func getCluster(service *SubnetService) string {
 }
 
 func (service *SubnetService) BuildSubnetID(subnet *v1alpha1.Subnet) string {
-	return util.GenerateIDByObject(subnet)
+	return util.GenerateIDByShortUID(subnet)
 }
 
 func (service *SubnetService) buildSubnetSetID(subnetset *v1alpha1.SubnetSet, index string) string {
-	return util.GenerateIDByObjectWithSuffix(subnetset, index)
+	return util.GenerateIDByShortUIDWithSuffix(subnetset, index)
 }
 
 // buildSubnetName uses format "subnet.Name_subnet.UUID" to ensure the Subnet's display_name is not
```

Comment:
Could you also update the comments for buildSubnetName/buildSubnetSetName to reflect that the display name will also be truncated?